src/app/forms/custom-sections/body/body.component.ts

```diff
@@ -68,7 +73,24 @@ export class BodyComponent implements OnInit, OnDestroy {
     return this.optionsService.getOptions(ReferenceDataResourceType.BodyMake);
   }
 
+  get dtpNumbers$(): Observable<MultiOptions> {
+    return combineLatest([
+      this.referenceDataService.getAll$(ReferenceDataResourceType.PsvMake),
+      this.referenceDataService.getReferenceDataLoading()
```

Comment:
Now that this works and intended. I would be nicer to use a loading flag specifically for the entity being used. i.e. `PsvMake`.